crates/cairo-profiler/src/trace_reader.rs

```diff
@@ -31,12 +31,12 @@ pub struct Sample {
 }
```

Comment:
Sample should either hold a single sample unit, or a map unit value. What we do now is leaking the abstraction.